trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -180,6 +180,26 @@ describe('controllers - case', () => {
         expect(res.redirect).toHaveBeenCalledWith('/not-found');
       });
     });
+
+    describe('should catch any thrown error', () => {
```

Comment:
This should probably be 

```suggestion
    describe('when an error is thrown', () => {
```

to be in keeping with the naming of describe blocks in this file?